### DISCUSSION TITLE
Search for Assets by Filename

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#17381] [Q&A / Support]
Is there any way to search for assets solely by filename? I tried the asset search feature:
![image](https://github.com/user-attachments/assets/c74e711f-d8ea-47f4-9e08-4091afabc4a0)
but it seems to search well beyond the filename itself, to include metadata and perhaps the contents of the files themselves.

**Comment by markus-moser** [comment]
You can use the asset grid view, add the filename column there and use the filter:

![image](https://github.com/user-attachments/assets/f1d9697a-f073-4b54-81bb-8f40d766f9b3)

**Reply by cancan101** [reply]
Gotcha. Is that the best / only way to do this search?

Annoyingly that search seems to be case sensitive: https://github.com/pimcore/admin-ui-classic-bundle/pull/389.

**Reply by markus-moser** [reply]
Yes that's currently the best option and yes it is case sensitive.
